mmpose/datasets/datasets/top_down/topdown_onehand10k_dataset.py

```diff
@@ -0,0 +1,231 @@
+import copy as cp
+import os
+import os.path as osp
+from collections import OrderedDict
+
+import json_tricks as json
+import numpy as np
+
+from mmpose.datasets.builder import DATASETS
+from .topdown_base_dataset import TopDownBaseDataset
+
+
+@DATASETS.register_module()
+class TopDownOneHand10KDataset(TopDownBaseDataset):
+    """OneHand10K dataset for top-down hand pose estimation.
+
+    The dataset loads raw features and apply specified transforms
+    to return a dict containing the image tensors and other information.
+
+    OneHand10K keypoint indexes::
+
+        0: 'wrist',
+        1: 'thumb1',
+        2: 'thumb2',
+        3: 'thumb3',
+        4: 'thumb4',
+        5: 'forefinger1',
+        6: 'forefinger2',
+        7: 'forefinger3',
+        8: 'forefinger4',
+        9: 'middle_finger1',
+        10: 'middle_finger2',
+        11: 'middle_finger3',
+        12: 'middle_finger4',
+        13: 'ring_finger1',
+        14: 'ring_finger2',
+        15: 'ring_finger3',
+        16: 'ring_finger4',
+        17: 'pinky_finger1',
+        18: 'pinky_finger2',
+        19: 'pinky_finger3',
+        20: 'pinky_finger4'
+
+    Args:
+        ann_file (str): Path to the annotation file.
+        img_prefix (str): Path to a directory where images are held.
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def __init__(self,
+                 ann_file,
+                 img_prefix,
+                 data_cfg,
+                 pipeline,
+                 test_mode=False):
+
+        super().__init__(
+            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+
+        self.ann_info['flip_pairs'] = []
+
+        self.ann_info['use_different_joints_weight'] = False
```

Comment:
not specific to this pr, but this name reads strange. maybe "use_different_joint_weights" is better? btw what does this parameter mean @jin-s13 

![image](https://user-images.githubusercontent.com/9464825/89410825-01c47400-d757-11ea-9709-d13d7cf0c6e4.png)

Reply:
aha. Yes.  `use_different_joint_weights` seems better. 
This will assign different loss weight to different joints (for example, pay more attention to the accuracy of wrists.).